futaba/cogs/filter/__init__.py

```diff
@@ -17,11 +17,24 @@
 from .core import Filtering
 from .filter import Filter
 
-
+# Setup for when cog is loaded
 def setup(bot):
+    setup_Filtering(bot)
+
+
+def setup_Filtering(bot):
```

Comment:
why are they named `setup_CogName`? not super pythonic in naming, a `.lower()` could be applied to make it `snake_case`

Reply:
As that is the way the cogs are named so I wanted to keep it that way so they matched. We can change it to make them snake case. It shouldn't affect anything, entirely up to you if we change it